MBBSEmu/HostProcess/HostRoutines/MenuRoutines.cs

```diff
@@ -298,6 +315,11 @@ private void MainMenuInput(SessionBase session, Dictionary<string, MbbsModule> m
             if (inputValue == 'X')
             {
                 session.SessionState = EnumSessionState.ConfirmLogoffDisplay;
+                //Load File if specified in appsettings.json and display if it exists
+                var ANSILogoffFileName = _configuration["ANSI.Logoff"];
```

Comment:
We should move this to after a user confirms logging of, replacing the default text `Ok, thanks for calling, etc.` in `LoggingOffDisplay()`